.infrastructure/hpa.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+ name: todoapp
+ namespace: mateapp
+spec:
+ scaleTargetRef:
+   apiVersion: apps/v1
+   kind: Deployment
+   name: todoapp
+ minReplicas: 2
+ maxReplicas: 5
+ metrics:
+ - type: Resource
+   resource:
+     name: cpu
+     target:
+       type: Utilization
+       averageUtilization: 70
+ - type: Resource
+   resource:
+     name: memory
+     target:
+       type: Utilization
+       averageUtilization: 70
+       
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_5_working_with_deployments/assets/87720583/e5fb3c19-23bb-45f3-b1df-c4644ae21168)
Eof doesn't work, because in last line is spase